docs/UsersGuide/source/Tutorial.rst

```diff
@@ -0,0 +1,589 @@
+
+   * ``*`` means CIN <= -500 J/kg
+   * ``+`` means -500 < CIN <= -250 J/kg
+   * ``/`` means -250 < CIN <= -100 J/kg
+   * ``.`` means -100 < CIN <= -25 J/kg
+
+In general, the higher the CIN values are (i.e., the closer they are to zero), the lower the convective inhibition and the greater the likelihood that a storm will develop. Low CIN values (corresponding to high convective inhibition) make it unlikely that a storm will develop even in the presence of high CAPE. 
+
+At the 0th forecast hour, the ``test_expt`` plot (below, left) shows lower values of CAPE and higher values of CIN than in the ``control`` plot (below, right). This means that ``test_expt`` is projecting lower potential energy available for a storm but also lower inhibition, which means that less energy would be required for a storm to develop. The difference between the two plots is particularly evident in the southwest corner of the difference plot, which shows a 1000+ J/kg difference between the two plots. 
+
+.. figure:: _static/plots/sfcape_diff_regional_f000.png
+      :width: 1200
+      :align: center
+
+      *CAPE/CIN Difference Plot at f000*
+
+At the 6th forecast hour, both ``test_expt`` and ``control`` plots are forecasting higher CAPE values overall. Both plots also predict higher CAPE values to the southwest of Indianapolis than to the northeast. This makes sense because the storm was passing from west to east. However, the difference plot shows that the ``control`` forecast is predicting higher CAPE values primarily to the southwest of Indianapolis, whereas ``test_expt`` is projecting a rise in CAPE values throughout the region. The blue region of the difference plot indicates where ``test_expt`` predictions are higher than the ``control`` predictions; the red/orange region shows places where ``control`` predicts significantly higher CAPE values than ``test_expt`` does. 
+
+.. figure:: _static/plots/sfcape_diff_regional_f006.png
+      :width: 1200
+      :align: center
+
+      *CAPE/CIN Difference Plot at f006*
+
+At the 12th forecast hour, the ``control`` plot indicates that CAPE may be decreasing overall. ``test_expt``, however, shows that areas of high CAPE remain and continue to grow, particularly to the east. The blue areas of the difference plot indicate that ``test_expt`` is predicting higher CAPE than ``control`` everywhere but in the center of the plot. 
+
+.. figure:: _static/plots/sfcape_diff_regional_f012.png
+      :width: 1200
+      :align: center
+
+      *CAPE/CIN Difference Plot at f012*
+
+Try It!
+----------
+
+Option 1: Adjust frequency of forecast output.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For a simple extension of this tutorial, users can adjust ``LBC_SPEC_INTVL_HRS`` and ``PLOT_FCST_INC`` to a factor of 12 less than 6 (which is the default). This will produce forecast output and plots at more frequent intervals for the duration of the 12-hour forecast window. 
```

Comment:
1) "to a factor of 12 less than 6" -- unclear what this means.  Are you resetting `LBC_SPEC_INTVL_HRS` or `PLOT_FCST_INC``, or both?
2) `LBC_SPEC_INTVL_HRS` specifies the time interval with which the LBCs are updated; it doesn't affect the frequency with which output files are created (last I checked).  I'm not familiar with `PLOT_FCST_INC` so can't comment about that (does it affect just the plotting or also the actual post-processed output interval?).

Reply:
@gsketefian I meant both, but you're right that the output is produced per hour anyway, so `LBC_SPEC_INTVL_HRS` shouldn't matter on a technical level. Do you think there's any value in decreasing `LBC_SPEC_INTVL_HRS` anyway? It seems like more frequent LBC updates would affect the forecast, and users could take a look at what changes. Or should I just leave that out? 
Increasing `PLOT_FCST_INC` would just allow users to view more frequent plots (e.g., a f03, f06, f09, f12) for a more fine-grained view of how the forecast is evolving visually. It shouldn't affect the forecast output though.

Reply:
@gspetro-NOAA I left a comment in the section above on `PLOT_FCST_INC` that I hope is helpful.  There is value in decreasing `LBC_SPEC_INTVL_HRS` since it will give you a better forecast (you're updating your LBCs more often) and assuming your external model can provide LBCs at that frequency.  Users can analyze that for sure.  Whether it will help the forecast more or less than some other change like using a different physics suite probably depends on the specific case being analyzed, so it's hard to say in advance.  But the larger your domain is, the less the LBCs are going to affect things.  If you do talk about changing `LBC_SPEC_INTVL_HRS`, I suggest finding an example in which changing it makes a significant difference.  Otherwise, I suggest leaving it out (we can have a whole section dedicated to this later on).  Right now, the output files are hard-coded to be hourly, so definitely talk about reducing `PLOT_FCST_INC` to plot output more often (but change the title of this section to something like "Adjust frequency of forecast **plots**" (not output).